extensions-core/stats/src/main/java/org/apache/druid/query/aggregation/variance/VarianceBufferAggregator.java

```diff
@@ -238,7 +238,9 @@ public void aggregate(ByteBuffer buf, int position)
         buf.putDouble(position + NVARIANCE_OFFSET, holder2.nvariance);
         return;
       }
-
+      if (holder2.count == 0) {
```

Comment:
I think this early return could happen right after `Preconditions.checkState(holder2 != null);` on line 233

Reply:
Good catch. Done.